### DISCUSSION TITLE
beta测试问题反馈

### DISCUSSION THREAD

**Discussion by ChingCdesu** [MaaAssistantArknights/MeoAsstElectronUI#95] [Q&A]
这里是electron ui的测试问题反馈帖，你可以在这里发布：
- 使用maa electron ui的beta版本遇到的bug，问题
- 提出改进建议，新功能提案等

**Comment by U98de** [comment]
写了半个多小时的反馈issue好像因为仓库改名没了？）GitHub你还我issue  o(TヘTo)
在这边简单说一下吧  o(TヘTo)~~（还好有粘贴板图片还在）~~
功能需求：
1.多设备执行任务的时候不能直观看到所有设备的进度
2.proxy
3.一键打开log文件夹
4.一个设备完成后就显示”所有已完成“
![image](https://user-images.githubusercontent.com/91559690/233447330-ae4d73a6-beb2-4d0f-aecc-0b6a9c663d10.png)
推荐换成这种形式
![image](https://user-images.githubusercontent.com/91559690/233447292-1609934b-458a-4685-99a9-b64fc33fb3f7.png)


bug：（所有bug都是在同时运行两个模拟器下发现的，未在单模拟器复刻）
1.任务栏图标 #101 
2.#102 补充：虽然备注和连接地址没变，但右边的任务列表确实变了，但没试点击开始后会怎么样
3.连续点击开始、停止会显示错误（这里出现过好几次不同的变种，但是我log已经删了）
![image](https://user-images.githubusercontent.com/91559690/233445737-bf6b340f-3e0d-4956-a0f0-0ee44ccb435d.png)
这里是在开始状态下开启关闭的任务后点击停止、开启后出现的

![image](https://user-images.githubusercontent.com/91559690/233445756-19ba9cd4-9008-4de3-a263-a653c662c056.png)
![image](https://user-images.githubusercontent.com/91559690/233445782-10b238df-6490-4a64-86c9-599b6c58a60d.png)
这里是时间同步增加，~~忘了怎么复现了~~

5.Task Group相关
 由#1切换到#2在切回#1就会显示错误，但是返回全部设备界面再重新进就会恢复（task group不能改名吗）
![image](https://user-images.githubusercontent.com/91559690/233445344-a8f84df9-4333-49a4-98a1-cd04b7fa0378.png)

6.最大化后再次点击无法返回原始尺寸

**Reply by pinaZC** [reply]
改进建议：
1.win11用户，希望主界面边框可以选择变为圆角
2.设置里添加本项目github地址，方便反馈（开发者那里可以暂时放个跳转链接吧）
3.首次运行没有引导，建议这里加一条：首次运行可点击左侧刷新按钮自动添加可连接设备
![image](https://user-images.githubusercontent.com/72682556/233999325-d70ff1e8-46b7-4fd9-a4a5-410cb821f4bc.png)
4.还是引导问题，这里没有按钮或者文字提示，根本不知道能点击装组件
![image](https://user-images.githubusercontent.com/72682556/233999649-296ef45e-019a-47b6-b0f0-a8a5016645e8.png)
bug：（可能也是改进建议）
删除不需要的任务后，下次启动客户端时会自动恢复，并且开关也会重置
![image](https://user-images.githubusercontent.com/72682556/233999978-60b3d7a9-222e-478e-b901-23c5bffd5a87.png)
重启软件后↓
![image](https://user-images.githubusercontent.com/72682556/234000530-6559c068-7e7a-4fe4-8a7c-2e43d1eee444.png)

**Comment by minecraftfen** [comment]
旧版的日志功能会在运行时告诉我现在运行了几次之类的信息，新版全部照搬没必要，但我觉得可以做个实时更新的信息栏。
还有，有关UI我有些改进的想法，但没法简单表达，你们这边接受Pull Request吗？以及，有没有一些文档之类的？

**Reply by minecraftfen** [reply]
话说...要不要试试看像GIMP一样把窗口拆碎，但留一个窗口合并(iframe应该够用)的选项给不习惯的用户。这样用户可以收起暂时不需要的窗口，只留下信息栏之类的。而且UI彻底模块化也有助于自定义和代码可维护性
